dnf-behave-tests/features/microdnf/install-xml-base.feature

```diff
@@ -0,0 +1,73 @@
+  And I execute "createrepo_c --baseurl file://{context.dnf.installroot}/xml_base/dnf-ci-fedora /{context.dnf.repos[dnf-ci-fedora].path}"
+  And I copy directory "{context.dnf.repos[dnf-ci-fedora].path}" to "/xml_base/dnf-ci-fedora"
+ When I execute microdnf with args "install setup"
+ Then the exit code is 0
+  And stderr is empty
+  And file "/xml_base/dnf-ci-fedora/noarch/setup-2.12.1-1.fc29.noarch.rpm" exists
+
+
+@bz1734350
+Scenario: Install packages from local repodata with xml:base pointing to remote packages
+#3. local repo with remote packages (different package location specified using xml:base)
+Given I make packages from repository "dnf-ci-fedora" accessible via http
+  And I copy repository "dnf-ci-fedora" for modification
+  And I execute "createrepo_c --baseurl http://localhost:{context.dnf.ports[dnf-ci-fedora]} /{context.dnf.repos[dnf-ci-fedora].path}"
+  And I use repository "dnf-ci-fedora"
+ When I execute microdnf with args "install setup"
+ Then the exit code is 0
+  And stderr is empty
+
+
+@bz1734350
+Scenario: Install packages from local repodata that have packages with xml:base pointing to a remote as well as local packages
+#4. local repo with local and remote packages; installing both at the same time.
+Given I make packages from repository "dnf-ci-fedora" accessible via http
+  And I copy repository "dnf-ci-fedora" for modification
+  And I copy repository "dnf-ci-thirdparty" for modification
+  And I execute "createrepo_c --baseurl http://localhost:{context.dnf.ports[dnf-ci-fedora]} /{context.dnf.repos[dnf-ci-fedora].path}"
+  And I execute "mergerepo_c --repo file://{context.dnf.repos[dnf-ci-fedora].path} --repo file://{context.dnf.repos[dnf-ci-thirdparty].path}" in "{context.dnf.installroot}"
+  And I configure a new repository "merged-repo" with
+      | key     | value                                        |
+      | baseurl | file://{context.dnf.installroot}/merged_repo |
+ When I execute microdnf with args "install setup alternator"
+ Then the exit code is 0
+  And stderr is empty
+
+
+@bz1734350
+Scenario: Install packages from remote repodata with remote packages
+#5. remote repo with remote packages
+Given I use repository "dnf-ci-fedora" as http
+ When I execute microdnf with args "install setup"
+ Then the exit code is 0
+  And stderr is empty
+
+
+@bz1734350
+Scenario: Install packages from remote repodata with xml:base pointing to packages on different remote
+#6. remote repo with remote packages (different package location (different url) specified using xml:base)
+Given I make packages from repository "dnf-ci-fedora" accessible via http
+  And I copy repository "dnf-ci-fedora" for modification
+  And I execute "createrepo_c --baseurl http://localhost:{context.dnf.ports[dnf-ci-fedora]} /{context.dnf.repos[dnf-ci-fedora].path}"
+  And I use repository "dnf-ci-fedora" as http
+ When I execute microdnf with args "install setup"
+ Then the exit code is 0
+  And stderr is empty
```

Comment:
Since you are writing quite extensive tests for the microdnf install command, may I ask you if you could add a test for the `--nodocs` option? I need the tests for a bug of mine - the simple install test you're adding is actually enough, but it's better to add a test for the option as well just to make sure it's covered...

Reply:
OK https://bugzilla.redhat.com/show_bug.cgi?id=1769831